integration-tests/contract-batch-smart-rollup-originate.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { CONFIGS } from './config';
+import { ligoSample } from './data/ligo-simple-contract';
+import { Protocols } from '@taquito/taquito';
+import { PvmKind } from '@taquito/rpc';
+
+CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
+  const Tezos = lib;
+  const mumbaiAndAlpha = protocol === Protocols.PtMumbai2 || protocol === Protocols.ProtoALpha ? test : test.skip;
+
+  describe(`Test contract.batch with smart rollup originate using: ${rpc}`, () => {
+    beforeEach(async (done) => {
+      await setup(true);
+      done();
+    });
+
+    mumbaiAndAlpha('should be able to batch smart rollup originate with other operations', async (done) => {
```

Comment:
could you change instances of `mumbaiAndAlpha` back into `it()`, otherwise everything else looks good

Reply:
right this was added after the change to tests. will do